burger/src/components/Main/index.js

```diff
@@ -0,0 +1,74 @@
+import "./main.css";
+import bacon from "../../assets/bacon.png";
+import cheese from "../../assets/cheese.png";
+import lettuce from "../../assets/lettuce.png";
+import meat from "../../assets/meat.png";
+import pickles from "../../assets/pickles.png";
+
+const Main = () => {
+  return (
+    <div className="main">
+      <div className="left__aside">
+        <div>
+          <h2>Our Price</h2>
+          <p>Bacon: 0.75 UAH</p>
+          <p>Cheese: 1.7 UAH</p>
+          <p>Salad: 0.95 UAH</p>
+          <p>Pickle: 1.25 UAH</p>
+          <p>Meat: 2 UAH</p>
```

Comment:
Better to save price in an array. It is much easier to maintain and update them later

Reply:
And use components, for example, `<Prices />`